### ISSUE TITLE
Can't install CollectionActions2 

### ISSUE THREAD

**Issue by solo-1234** [alexed1/LightningFlowComponents#362]
Hi! I'm getting an error when trying to install Collection Actions for flow - https://unofficialsf.com/list-actions-for-flow/

The error is:
```
Cannot add component of type:ApexClass named:SearchUtilsTest subjectId:01p55000001zvQM to another package because it is an installed component.
package.xml 
```
It seems that the SearchUtilsTest class is there since I installed an older version of the package, which was called CollectionProcessors (04tf4000004Psnh).

Any idea how to fix it? Should I just delete the old version of the class and then try installing again?

Thank you

**Comment by alexed1** [comment]
Yes, we try to keep the amount of interface change to a minimum to prevent conflicts like this. See if you can just delete the old version. the trickiness comes if you used it in a ton of flows. If necessary, we'll create a managed package version of this which won't interfere with anything.

**Comment by solo-1234** [comment]
Thanks for the quick response! Should I try deleting the whole package, or just the class(es) that cause the problem?

**Comment by solo-1234** [comment]
It's not letting me uninstall the package because it's used by a bunch of flows :(

**Comment by solo-1234** [comment]
The version I have now is 1.13. Is there an easy way for me to check which files have changed since then, and I can copy/paste to update them manually? I think that might be easier.
